src/components/MapControls/MapControls.js

```diff
@@ -40,13 +43,102 @@ const degreesToRadians = (degrees) => {
   return degrees * (Math.PI / 180);
 };
 
+const useStyles = makeStyles((theme) => {
+  return {
+    mapControls: {
+      position: "absolute",
+      display: "flex",
+      flexDirection: "column",
+      gap: 8,
```

Comment:
should be also 10 on mobile and 15 on desktop

Reply:
ok